--- baremetal.cfg	2020-09-08 18:01:40.329789200 +0300
+++ baremetal.cfg	2020-09-08 18:20:56.120775400 +0300
@@ -35,6 +35,11 @@
 d-i partman/mount_style select traditional
 d-i partman-auto/expert_recipe string                         \
       single-root-no-swap ::                                  \
+              300 300 300 free                                \
+                      $iflabel{ gpt }                         \
+                      $reusemethod{ }                         \
+                      method{ efi }                           \
+                      format{ } .                             \
               1000 50 -1 ext4                                 \
                       filesystem{ ext4 }                      \
                       method{ format } format{ }              \
